apis/nucleus/src/components/listbox/assets/__tests__/get-list-sizes.test.jsx

```diff
@@ -90,6 +93,64 @@ describe('get-list-sizes', () => {
     });
   });
 
+  it('grid mode with layoutOrder column and frequency activated should be wider', () => {
+    args.layout.layoutOptions.dataLayout = 'grid';
+    args.layout.layoutOptions.layoutOrder = 'column';
+    args.freqIsAllowed = true;
+    const sizes = getListSizes(args);
+    expect(sizes).toEqual({
+      columnCount: 34,
+      columnWidth: 108,
+      count: 200,
+      itemPadding: 4,
+      itemSize: 36,
+      listCount: 100,
+      listHeight: 300,
+      frequencyWidth: 40,
+      maxCount: {
+        column: 310648,
+        row: 577000,
+      },
+      overflowStyling: {
+        overflowY: 'hidden',
+      },
+      rowCount: 3,
+      scrollBarWidth: 10,
+    });
+  });
+
+  it('A minimum item width should kick in if text is short', () => {
+    args.layout.layoutOptions.dataLayout = 'grid';
+    args.layout.layoutOptions.layoutOrder = 'column';
+    args.textWidth = 10;
+    args.textWidth = 10;
+    const MIN_WIDTH = 56;
+    const sizes = getListSizes(args);
+    expect(sizes).toMatchObject({
+      columnWidth: MIN_WIDTH,
+      maxCount: {
+        column: 599107,
+      },
+    });
+  });
+
+  it('A minimum item width should kick in if text is short and reverve extra space for frequency', () => {
```

Comment:
"reverve" -> reserve, just a typo

Reply:
I will change in a subsequent PR if I remember 😄